src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -406,8 +432,8 @@ public static IServiceCollection AddApplicationInsightsTelemetryProcessor<T>(thi
 
         private static bool IsApplicationInsightsAdded(IServiceCollection services)
```

Comment:
what is this method used for? Why the change?

Reply:
we use this to detect if AI services are already added to DI to prevent double injection if customer calls both AddApplicationInsightsTelemetry() and UseApplicationInsights(). We previously relied on presence of 'ApplicationInsightsDebugLogger' to detect this. With that no longer needed (since ilogger is automatically enabled), we are now relying on presense of telemetryclient to detect if AI is already added.